adapters/taboola/taboola.go

```diff
@@ -149,7 +146,7 @@ func (a *adapter) buildRequest(request *openrtb2.BidRequest) (*adapters.RequestD
 
 // Builds endpoint url based on adapter-specific pub settings from imp.ext
 func (a *adapter) buildEndpointURL(publisherId string, mediaType string) (string, error) {
-	endpointParams := macros.EndpointTemplateParams{PublisherID: publisherId, MediaType: mediaType, Host: a.hostName}
+	endpointParams := macros.EndpointTemplateParams{PublisherID: publisherId, MediaType: mediaType, Host: a.gvlID}
```

Comment:
Should not assign `a.gvlID` value to host. Instead consider adding `GvlID` field in `macros.EndpointTemplateParams` struct. 

With above change, also update `endpoint` to use `{{.GvlID}}` instead of `{{.Host}}` . This is needed to correctly resolve macros

https://github.com/prebid/prebid-server/blob/4109b905c292d1f43c44e15a9951b5e7ec22bb92/static/bidder-info/taboola.yaml#L1

Reply:
done